cmd/xl-sets.go

```diff
@@ -823,11 +823,13 @@ func (f *FileInfoCh) Push(fi FileInfo) {
 
 // Calculate least entry across multiple FileInfo channels, additionally
 // returns a boolean to indicate if the caller needs to call again.
-func leastEntry(entriesCh []FileInfoCh, totalDrives int, heal bool) (FileInfo, bool) {
-	var entriesValid = make([]bool, len(entriesCh))
-	var entries = make([]FileInfo, len(entriesCh))
-	for i := range entriesCh {
-		entries[i], entriesValid[i] = entriesCh[i].Pop()
+func leastEntry(entryChs []FileInfoCh, heal bool, entries []FileInfo, entriesValid []bool) (FileInfo, int, bool) {
```

Comment:
Please explain what the returned integer is. It seems to be the quorum count, but a comment would be nice for the future.

Reply:
Yes will add a comment